107/build.gradle

```diff
@@ -37,24 +39,25 @@ dependencies {
   }
 }
 
-javadoc {
-  exclude '**/tck/**'
+test {
+  if (JavaVersion.current().isJava9Compatible()) {
+    jvmArgs 'add-modules', 'java.xml.bind'
```

Comment:
To be clear, when you started this, you added `addmods`, the former version of this flag